tests/components/yeelight/test_config_flow.py

```diff
@@ -230,32 +234,55 @@ async def test_options(hass: HomeAssistant):
         await hass.async_block_till_done()
 
     config = {
+        CONF_NAME: NAME,
         CONF_MODEL: "",
         CONF_TRANSITION: DEFAULT_TRANSITION,
         CONF_MODE_MUSIC: DEFAULT_MODE_MUSIC,
         CONF_SAVE_ON_CHANGE: DEFAULT_SAVE_ON_CHANGE,
         CONF_NIGHTLIGHT_SWITCH: DEFAULT_NIGHTLIGHT_SWITCH,
     }
-    assert config_entry.options == {
-        CONF_NAME: "",
-        **config,
-    }
+    assert config_entry.options == config
     assert hass.states.get(f"light.{NAME}_nightlight") is None
 
     result = await hass.config_entries.options.async_init(config_entry.entry_id)
     assert result["type"] == "form"
     assert result["step_id"] == "init"
 
     config[CONF_NIGHTLIGHT_SWITCH] = True
+    user_input = {**config}
+    user_input.pop(CONF_NAME)
     with patch(f"{MODULE}.Bulb", return_value=mocked_bulb):
         result2 = await hass.config_entries.options.async_configure(
-            result["flow_id"], config
+            result["flow_id"], user_input
         )
         await hass.async_block_till_done()
     assert result2["type"] == "create_entry"
-    assert result2["data"] == {
-        CONF_NAME: "",
-        **config,
-    }
+    assert result2["data"] == config
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)